server/services/transactionsTransformer.go

```diff
@@ -416,5 +431,56 @@ func (transformer *transactionsTransformer) addOperationsGivenTransactionEvents(
 		rosettaTx.Operations = append(rosettaTx.Operations, operations...)
 	}
 
+	for _, event := range eventsESDTNFTCreate {
+		if !transformer.provider.HasCustomCurrency(event.identifier) {
+			// We are only emitting balance-changing operations for supported currencies.
+			continue
+		}
+
+		operations := []*types.Operation{
+			{
+				Type:    opCustomTransfer,
+				Account: addressToAccountIdentifier(event.otherAddress),
+				Amount:  transformer.extension.valueToCustomAmount(event.value, event.getExtendedIdentifier()),
+			},
+		}
+
+		rosettaTx.Operations = append(rosettaTx.Operations, operations...)
```

Comment:
Delayed a bit, since there's extra non-common logic to be handled (e.g. sign of `amount`, the address to be used as impacted `account`).

Is this all right?

Reply:
ok